winpr/libwinpr/utils/wlog/wlog.c

```diff
@@ -716,6 +732,9 @@ void WLog_Uninit()
 	wLog* child = NULL;
 	wLog* root = WLog_GetRoot();
 
```

Comment:
I'd suggest to use g_RootLog instead of WLog_GetRoot() which might create a new root log which is not really what we want in uninit ;)